apis/Google.LongRunning/Google.LongRunning/OperationsClientPartial.cs

```diff
@@ -34,6 +35,19 @@ public virtual IScheduler Scheduler
         {
             get { throw new NotImplementedException(); }
         }
+
+        // It would be nice to make this internal, but then anyone creating their
+        // own subclass for testing would be broken.
```

Comment:
Hmm... protected internal should work, so long as mocking libraries are able to cope with it. Basically, we want users to be able to test with their own code, so they've got to be able to override this to avoid polling operations going bang.